tests/Psr3EveryLoggerTest.php

```diff
@@ -97,4 +110,12 @@ public function testLogUnknownLogLevelYii2()
         $this->expectException('\Psr\Log\InvalidArgumentException');
         $logger->log('unknown', 'A test message');
     }
-}
\ No newline at end of file
+
+    public function testArrayYii2()
+    {
+        $logger = new Psr3Yii2Logger();
+        $errorMessages = ['Error Message 1', 'Error Message 2'];
```

Comment:
This is a subtle thing, but the array syntax is (as I understand it) for logging structured error data, not multiple different errors. This test obviously works, but implies that the array is for logging multiple errors, which could be misleading.

```suggestion
        $errorMessages = ['message' => 'There was a problem', 'details' => 'More info'];
```

Reply:
I didn't even see your suggestion. I went a little more complicated.